docs/reference/queries.txt

```diff
@@ -1900,23 +1898,9 @@ not possible:
     default_scope ->{ where('tags.foo' => 'bar') }
```

Comment:
This should have an example showing what the result of `Band.new` is, would it have the `tags.foo` as the attribute name?

Reply:
I did not realize this would be a thing... I added docs and tests that demonstrate this.